devkit/samples/examples-tables-undefined/examples-undefined.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Examples Tables - With Undefined Steps
+  The replacement pattern used in scenario outlines does not influence how steps
+  are matched. The replacement pattern is replaced, and step definitions are
+  matched against that text. Because of that the following results in 3
+  undefined steps and 3 suggestions to implement each step. 
```

Comment:
Shouldn't this only yield 1 suggested example for each step.

i.e. scenario one only has 1 "bad" step, same for scenario 2/3